files/es/learn/css/howto/css_faq/index.md

```diff
@@ -13,22 +13,18 @@ Los navegadores modernos tienen fundamentalmente dos modos de renderizado:
 - _Quirks mode_ (Modo no estándar): también se llama el modo de compatibilidad con versiones anteriores y permite que las páginas web heredadas se representen como sus autores habían previsto, siguiendo las normas de representación o renderizado no estándares que usan los navegadores antiguos. Los documentos con una declaración `DOCTYPE` incompleta, incorrecta o faltante o con una declaración `DOCTYPE` conocida que se usara habitualmente antes de 2001 se representarán en el Modo no estándar.
 - _Standards Mode_ (Modo estándar): el navegador intenta seguir estrictamente los estándares del W3C. Se supone que las nuevas páginas HTML se diseñarán para navegadores compatibles con los estándares, y como resultado, las páginas con una declaración `DOCTYPE` moderna será renderizada con el Modo estándar.
 
-Los navegadores basados en Gecko tienen un tercer _[Modo casi estándar](/en/Gecko's_"Almost_Standards"_Mode)_ que tiene solo una peculiaridades menores.
+Los navegadores basados en Gecko tienen un tercer _[Modo casi estándar](/en/Gecko's_"Almost*Standards"\_Mode)* que tiene solo una peculiaridades menores.
```

Comment:
```suggestion
Los navegadores basados en Gecko tienen un tercer [_Modo casi estándar_](/es/docs/Mozilla/Gecko_Almost_Standards_Mode) que tiene solo una peculiaridades menores.
```

Reply:
Argh, Prettier bugs...so annoying.  💢

Reply:
If it's alright, I'd actually like to keep the underscores for italics outside of the link, and instead add this specific file to `.prettierignore` -- the main reason is because I want to build up a list of files that Prettier is incorrectly formatting so we can put some pressure on Prettier to fix this annoying bug!

Reply:
oh, great! I remember that in caugner's linting it was inside, but i take note to keep it outside

thank you!